package.json

```
@@ -8,10 +8,10 @@
 		"url": "git://github.com/joyent/node-jsprim.git"
 	},
 	"dependencies": {
-		"assert-plus": "1.0.0",
-		"extsprintf": "1.3.0",
-		"json-schema": "0.2.3",
-		"verror": "1.10.0"
+		"assert-plus": "^1.0.0",
+		"extsprintf": "^1.3.0",
+		"json-schema": "^0.2.5",
+		"verror": "^1.10.0"
```

I think this PR is important for downstream users and I hope the maintainers can review and merge. But perhaps the versions should remain pinned rather than allowing ranges?

```suggestion
		"assert-plus": "1.0.0",
		"extsprintf": "1.3.0",
		"json-schema": "0.2.5",
		"verror": "1.10.0"
```